### ISSUE TITLE
chinese  recognition

### ISSUE THREAD

**Issue by jiaozhusos** [SeanNaren/deepspeech.pytorch#281]
I want to ask whether  it can  deal with chinese datasets  and train ?  And  can the the pre-trained model   recognize   chinese  ?

**Comment by Wilbur529** [comment]
I tried Mandarin recognition by changing the way to load data and making an alternativa labels.json. But i haven't tried to use the pre-trained model.

**Comment by jiaozhusos** [comment]
thanks, by the way ,I also look  baidu- paddle- deepspeech2, if their data can be used? And if I  train the datasets by myself ,it takes approximately how long time?(no gpu ,only cpu........)

**Comment by Wilbur529** [comment]
sry, i am not familiar with PaddlePaddle/DeepSpeech Project, but i think if you have one set of wave files and there are correct labels corresponded with, then you can train a model. By the way, you know the model have many RNN layers, and lots of parameters, so if you have no gpu or enough RAM space, the program may got some mistakes.

**Comment by ryanleary** [comment]
It should work but I have not tested it.

**Comment by minushuang** [comment]
@Wilbur529 can u share your  Mandarin labels?I just want to train a Mandarin .thanks a lot.
